packages/plugin-ext/package.json

```diff
@@ -16,6 +16,7 @@
     "@theia/markers": "1.23.0",
     "@theia/messages": "1.23.0",
     "@theia/monaco": "1.23.0",
+    "@theia/monaco-editor-core": "alpha",
```

Comment:
Can we get away with making this a dev-dependency?

Reply:
Since `plugin-ext` depends on `@theia-monaco`, and `monaco-editor-core` wouldn't be a dev dependency there, I don't think making it a dev dependency here would actually have any effect? But perhaps I'm wrong.